Packs/Claroty/Classifiers/classifier-Claroty.json

```diff
@@ -1,187 +1,17 @@
 {
-	"brandName": "Claroty",
-	"custom": true,
+    "id": "Claroty",
+    "name": "Claroty - Classifier",
+    "type": "classification",
+    "description": "Classifies Claroty incidents.",
```

Comment:
![image](https://user-images.githubusercontent.com/7270217/82729487-a4488b80-9d00-11ea-807b-dbc53fd5114a.png)
fix the indentations